stdlib/public/stubs/LibcShims.cpp

```diff
@@ -50,7 +53,11 @@ SWIFT_RUNTIME_STDLIB_INTERFACE
 __swift_size_t swift::_swift_stdlib_fwrite_stdout(const void *ptr,
                                                   __swift_size_t size,
                                                   __swift_size_t nitems) {
+#if defined(__ANDROID__)
+  return __android_log_write(ANDROID_LOG_DEBUG, "SwiftRuntime", static_cast<const char *>(ptr));
```

Comment:
This is a good implementation for now given what you're trying to accomplish here, but it isn't quite correct in terms of what we're trying to with `fwrite` on other platforms.  `__android_log_write` is going to try to ask the buffer for its length - and we're just slicing a buffer pointer and handing that to the runtime call here.  You should insert the NUL byte in the right place before handing that off.